res/css/views/rooms/_EventTile.scss

```diff
@@ -1074,14 +1074,20 @@ $threadInfoLineHeight: calc(2 * $font-12px); // See: _commons.scss
     .mx_EventTile {
         // Override :not([data-layout="bubble"])
         &[data-layout=group] {
+            --MatrixChat_useCompactLayout-top-avatar: 2px;
+            --MatrixChat_useCompactLayout-top-e2eIcon: 3px;
+            --MatrixChat_useCompactLayout_line-spacing-block: 0px;
```

Comment:
This might look redundant but I think this should ensure that the same zero margin is applied to e2eIcon, avatar, line, and reply.